### ISSUE TITLE
report v2.0,v2.1 (main) is not working when server takes too long to respond

### ISSUE THREAD

**Issue by boly38** [boly38/action-umami-report#31]
following last merge, report is no more working

- 🔴  [(private) test job](https://github.com/boly38/action-umami-report-test/actions/runs/3112429002/jobs/5045839382) that target `main` (v2.0)
```
ERROR: Error: Could not get websites
Options: {}
Stacktrace:
Error: Error: Could not get websites
Options: {}
Stacktrace:
```
- ✅ local manual test is green

**Comment by boly38** [comment]
- A retry of the same job passed, first failure was maybe due to temp unavailability? strange empty Stacktrace
- (minor) need to remove default `*first*` domain value

**Comment by boly38** [comment]
- [ ] wait for umami-api-client new version : fix of jakobbouchard/umami-api-client#1
- [ ] remove default *first* domain value

**Comment by boly38** [comment]
Following previous fix #32 (v2.0.1) and new run, we have now root cause and this is related to umami api client default timeout which is 1 sec.

An enhancement has been created : jakobbouchard/umami-api-client#3

TODO:
- [ ] grab this enhancement and  update umami-api-client version 
- [ ] set default timeout to 5 seconds

**Comment by boly38** [comment]
commit: 773ad31256901452c86c50343b4d38a77c4b3529 (wrong commit desc issue number)
